### PR TITLE
misc: Update Codeowners for breeze

### PR DESCRIPTION
**Pull request by assignUser**
Breeze is an entire separate library that is not build in CI. I would like to not get notifications for the dir, the way to do this in CODEOWNERS is to specify the dir and explicitly add a CO (or none to clear).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -29,6 +29,9 @@ CMake/ @assignUser @majetideepak
 scripts/ @assignUser @majetideepak
 .github/ @assignUser @majetideepak
 
+# clear codeownership in these dirs
+velox/experimental/breeze
```
Comment (Yuhta) [inline .github/CODEOWNERS:33]: Let's add @dreveman as the owner

Comment (assignUser) [reply .github/CODEOWNERS]: It only works if you have write access to the repo: 
image

Comment (Yuhta) [reply .github/CODEOWNERS:33]: @pedroerp How do we grant write access?

Comment (pedroerp) [reply .github/CODEOWNERS:33]: @Yuhta @assignUser added. Could you try again?

Comment (assignUser) [reply .github/CODEOWNERS]: I assume @dreveman needs to accept the invite as it still shows an error but should work once it shows the 'collaborator' tag

Comment (dreveman) [reply .github/CODEOWNERS]: I just accepted it

**Review by dreveman — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (assignUser) [reply .github/CODEOWNERS]: :tada: 
image

**Review by Yuhta — APPROVED** [review]

**Pull request closed without merging**